packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -64,6 +64,14 @@ export const CallWithChatControlBar = (props: CallWithChatControlBarProps): JSX.
   const theme = useTheme();
   const callWithChatStrings = useCallWithChatCompositeStrings();
   const options = inferCallWithChatControlOptions(props.mobileView, props.callControls);
+  const chatButtonStrings = {
```

Comment:
nit: this should probably be memoized... but also you can ignore this as they're small buttons so rerenders aren't costly

Reply:
will look at this!

Reply:
Done in this PR: https://github.com/Azure/communication-ui-library/pull/1606